target/linux/econet/image/Makefile

```diff
@@ -0,0 +1,51 @@
+include $(TOPDIR)/rules.mk
+include $(INCLUDE_DIR)/image.mk
+
+define Target/Description
+	Build firmware images for EcoNet MIPS based boards.
+endef
+
+# tclinux-trx is the default format used in the SDK
+define Build/tclinux-trx
+  ./tclinux-trx.sh $@ $(IMAGE_ROOTFS) $(VERSION_DIST)-$(REVISION) > $@.new
+	mv $@.new $@
+endef
+
+# tclinux bootloader requires LZMA, BUT only provides 7.5MB of space
+# to decompress into. So we use vmlinuz and decompress twice.
+define Device/Default
+  DEVICE_DTS_DIR := ../dts
+  KERNEL_SIZE := 7480k
+  KERNEL_NAME := vmlinuz.bin
+  KERNEL_LOADADDR := 0x80020000
+  KERNEL := kernel-bin | append-dtb
+endef
+
+# NOTE: This will not work for upgrading from factory because it requires a cryptographic signature
+#       however, it it can be flashed, then it will boot correctly.
+define Device/tplink_archer-vr1200v-v2
+  DEVICE_VENDOR := TP-Link
+  DEVICE_MODEL := Archer vr1200v
+  DEVICE_VARIANT := v2
+  TPLINK_FLASHLAYOUT := 16Mmtk
+  TPLINK_HWID := 0x0b473502
+  TPLINK_HWREV := 0x0006007c
+  TPLINK_HWREVADD := 0x0
+  TPLINK_HVERSION := 3
+  DEVICE_DTS := en751221_tplink_archer-vr1200v-v2
+  IMAGES := sysupgrade.bin
+  IMAGE/sysupgrade.bin := append-kernel | lzma | pad-to 4193792 | append-rootfs | \
+    tplink-v2-header -R 0x400000
+endef
+TARGET_DEVICES += tplink_archer-vr1200v-v2
+
+define Device/smartfiber_xp8421-b
```

Comment:
Sorting. Smartfiber should be before tp-link :)

Reply:
Will fix.